src/app/zones/components/EntityScanner.vue

```diff
@@ -3,7 +3,7 @@
     <div class="scanner-content">
       <KEmptyState cta-is-hidden>
         <template #title>
-          <div class="mb-2">
+          <span class="mr-1">
```

Comment:
Can we start to try to avoid using these helper classes? At the very least it seems strange to me that sometimes we use these and sometimes we use the style node  (for example the margin-right below https://github.com/kumahq/kuma-gui/pull/1450/files#diff-3bc3c5b294b392b43bea19c02637aecd234549d4ce0dc204cb399f291eb33c3eR178).

Reply:
Why? They’re convenient. What’s strange about using the utility for situations where nothing else is required and a property declaration for when there already happens to be a CSS rule set?

Reply:
Well you've almost pointed out one reason yourself. If I want to add more styles, then there will already be a CSS ruleset to add to.

Also, it means styles are always defined in the same place, in style nodes not bits in classes and bits in style nodes.

Don't worry about it for now.